receiver/fluentforwardreceiver/conversion.go

```diff
@@ -135,8 +135,10 @@ func decodeTimestampToLogRecord(dc *msgp.Reader, lr pdata.LogRecord) error {
 	return nil
 }
 
-func parseRecordToLogRecord(dc *msgp.Reader, lr pdata.LogRecord) error {
+func parseRecordToLogRecord(dc *msgp.Reader, lr pdata.LogRecord, conf *Config) error {
 	attrs := lr.Attributes()
+	bodyMap := lr.Body().MapVal()
+	lr.SetName("fluent[d|bit]")
```

Comment:
Arbitrary name to provide something in name (I think otel collector should at least show the good exemple)